website/cue/reference/components/sinks/elasticsearch.cue

```diff
@@ -225,7 +253,7 @@ components: sinks: elasticsearch: {
 			}
 		}
 		endpoint: {
-			description: "The Elasticsearch endpoint to send logs to. This should be the full URL as shown in the example."
+			description: "The Elasticsearch endpoint/endpoints to send logs to. This should be the full URL as shown in the example."
 			required:    true
 			type: string: {
 				examples: ["http://10.24.32.122:9000", "https://example.com", "https://user:password@example.com"]
```

Comment:
We should add an example of multiple configured endpoints

Reply:
I don't see a way to have both strings and lists of strings in that examples list.

Any idea how to?

Reply:
😅 oh no cue... I tried a couple things locally and it all blew up. Maybe @jszwedko has an idea as the resident cue-xpert.